libmamba/src/core/transaction.cpp

```diff
@@ -1082,9 +1082,16 @@ namespace mamba
         }
         else
         {
+            // TODO: add executable name
+            // TODO: add environment name/prefix
             LOG_INFO << "Waiting for pyc compilation to finish";
             m_transaction_context.wait_for_pyc_compilation();
-            Console::stream() << "Transaction finished";
+            Console::stream() << "\nTransaction finished\n\n"
+                              << "To activate this environment, use:\n\n"
+                              << "    $ [micro]mamba activate <environment>\n\n"
+                              << "Or to execute a single command in this environment, use:\n\n"
+                              << "    $ [micro]mamba run -n <environment> mycommand\n";
```

Comment:
> This message is being triggered in all transactions, install, update, etc, and not only environment creation. It is a bit noisy that way b/c for most of those the user is already in the environment and defeats the purposes of the message.

Hmm I agree!

> PS: It would be nice to change the `<environment>` for the actual name of the environment, mimicking conda's behavior and making the message "copy-n-pastable."

It's already the case on the last version of the code :)

Reply:
> It's already the case on the last version of the code :)

Awesome. Waiting for the new release.